### PR TITLE
Build Event Form

### PR DESCRIPTION
**Pull request by iiio2**

### CODE REVIEW

File: pages/events/add.vue [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    <FormKit
+      name="name"
+      id="name"
+      label="Event Name"
+      validation="required|length:5"
+      min="5"
+      v-model="data.name"
+      :classes="{
+        input: {
+          'shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline': true,
```
Comment (razbakov) [inline pages/events/add.vue]: Make a theme, so that we don't need to define classes in each form field

File: package.json [hunk 2]
```diff
@@ -19,20 +19,24 @@
...
     "commander": "^10.0.0",
     "dotenv": "^16.0.3",
     "eslint": "^8.33.0",
     "fs": "^0.0.1-security",
-    "nuxt": "^3.2.3",
+    "nuxt": "3.5.2",
     "path": "^0.12.7",
+    "postcss": "^8.4.24",
     "prisma": "^4.12.0",
+    "tailwindcss": "^3.3.2",
```
Comment (razbakov) [inline package.json]: Tailwind is already installed and configured with nuxtjs/tailwind package

**Review by razbakov — CHANGES_REQUESTED** [review]

File: pages/events/add.vue (hunk 1 above)
Comment (iiio2) [reply pages/events/add.vue]: theme added.

File: package.json (hunk 2 above)
Comment (iiio2) [reply package.json]: removed new tailwind

File: formkit.config.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      }, 
+      date: {
+        input:'shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline', 
+      }, 
+      submit:{
+        input:'bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded focus:outline-none focus:shadow-outline'
+      }
+    })
+  }
+}
```
Comment (razbakov) [inline formkit.config.ts]: Add new line

File: tailwind.config.ts [hunk 4]
```diff
@@ -0,0 +1,6 @@
+import formKitTailwind from '@formkit/themes/tailwindcss'
+
+export default {
+  content: ['./src/**/*.{html,js}', './formkit.config.ts'],
+  plugins: [formKitTailwind],
+}
```
Comment (razbakov) [inline tailwind.config.ts]: Add new line

**Review by razbakov — CHANGES_REQUESTED** [review]

File: formkit.config.ts (hunk 3 above)
Comment (iiio2) [reply formkit.config.ts]: done

File: tailwind.config.ts (hunk 4 above)
Comment (iiio2) [reply tailwind.config.ts]: done

File: package-lock.json
Comment (razbakov) [inline package-lock.json:1]: It seems you used npm instead of yarn. We use yarn. Remove this file.

**Review by razbakov — CHANGES_REQUESTED** [review]

**Review by razbakov — CHANGES_REQUESTED** [review]
When I run `yarn dev` I am getting 500 error:
__vite_ssr_import_1__.ssrComplete is not a function

File: package-lock.json
Comment (iiio2) [reply package-lock.json:1]: solved

**Comment by iiio2** [comment]
@razbakov  i am not facing it. I tested it.

**Pull request closed without merging**